lib/infomon/activespell.rb

```diff
@@ -42,6 +42,10 @@ def self.get_spell_info(spell_check = XMLData.active_spells)
         makeychange << k
         update_spell_names.push('Next Bounty')
         next
+      when /(?:Resist Nature|620) (?:- (?:Heat|Cold) \(\d\d%|- Steam \(\d\d|- Lightning|\(\d\d%\))/
```

Comment:
Could the regex be simplified a bit to:
`(?:Resist Nature|620)(?: - (?:Heat|Cold|Steam|Lightning))? \(\d{2}%\)`

https://rubular.com/r/uuwlYBJsybDyF6

It was kinda weirding me out heat/cold got lumped together but then the rest had to be separate. 😨

Reply:
It's because of truncing in the XML due to character limits by Simu as to why they're separated out like that.